account/templates/account/settings.html

```diff
@@ -0,0 +1,14 @@
+{% extends "base.html" %}
+
+{% load static %}
+#bez tego tagu nie wyświetla image
```

Comment:
Could you move this tag to the `base` template? All comments should be written in English

Reply:
Good point, I will move it. This comment was information only for me (while learning). Now I can remove it.

Reply:
I just remembered... When this tag was only in the base template, images were not displayed. 
However, I can remove the tag, because this template does't contain images currently.